Plugins~/Src/MeshSyncClientBlender/msblenContext.cpp

```diff
@@ -905,8 +905,9 @@ void msblenContext::doExtractNonEditMeshData(msblenContextState& state, BlenderS
         blender::barray_range<mu::tvec3<float>> normals = bmesh.normals();
         if (!normals.empty()) {
             dst.normals.resize_discard(num_indices);
+            const float roundingFactor = 100.0f;
             for (size_t ii = 0; ii < num_indices; ++ii)
-                dst.normals[ii] = normals[ii];
+                dst.normals[ii] = ceil(normals[ii] * roundingFactor) / roundingFactor;
```

Comment:
Can you create a utility function for this in the msblenUtils class or something like that? Seems like this logic would be commonly re-used

Reply:
Done here https://github.com/unity3d-jp/MeshSync/pull/710